rock_paper_scissors/rps.py

```diff
@@ -3,8 +3,25 @@
 import sys
 
 def rock_paper_scissors(n):
-  pass 
+  #n stands for number of rounds
+  # 3 available options results in 3 ^ n for total no. of plays
 
+    outcomes = []#collect the combinations played
+    plays = ['rock','paper','scissors']
+    
+    #Logic to play the game
+    #In a round you can make simple plays of above stated, how it changes when you have two rouds
+    def game(plays_done, rounds_left_play):
+        if rounds_left_play == 0:#end case where rounds reach 0, do no more plays and break out
+            outcomes.append(plays_done)
+            return
+        for i in range(0,len(plays)):
+             # Add plays to each of plays_done, call the function itself again with less round
```

Comment:
Now we need to call our _____ function, modifying our rounds and result, coming closer to the base case.